tests/cupyx_tests/scipy_tests/special_tests/test_log_softmax.py

```diff
@@ -0,0 +1,65 @@
+import cupy
+from cupy import testing
+import cupyx.scipy.special  # NOQA
+
+import scipy.special  # NOQA
+
+
+atol = {'default': 1e-6, cupy.float64: 1e-14}
+rtol = {'default': 1e-6, cupy.float64: 1e-14}
+
+atol_low = {'default': 1e-6, cupy.float16: 1e-3, cupy.float64: 1e-14}
+rtol_low = {'default': 1e-6, cupy.float16: 1e-3, cupy.float64: 1e-14}
+
+
+@testing.with_requires('scipy')
+class TestLogSoftmax:
+
+    @testing.for_dtypes('bhilefdFD')
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=atol, rtol=rtol)
+    def test_log_softmax_ndarray_1(self, xp, scp, dtype):
+        a = xp.arange(100, dtype=dtype)
```

Comment:
Please properly adjust the input to avoid the overflow warning. I think `arange(1)` makes no sense as a test, and maybe `shaped_random` with properly scaled would work. (Also check the default value of `shaped_random`'s `scale` parameter)

Reply:
Even `np.exp(20)` makes inf in float16.
```
>>> np.exp(np.linspace(0, 100, num=10, dtype=np.int8))
array([1.000e+00, 5.987e+04,       inf,       inf,       inf,       inf,
             inf,       inf,       inf,       inf], dtype=float16)
```

Reply:
> I think arange(1) makes no sense as a test, and maybe shaped_random with properly scaled would work.

Agreed with you. I've fixed it. it Is this okay now?

Reply:
> EDIT 2: Updated the tests. Is it okay to separate tests?

Let's skip unsigned integer tests in one test, leaving a comment as https://github.com/cupy/cupy/pull/6823#pullrequestreview-1036798376